docs/installation.rst

```diff
@@ -31,8 +31,7 @@ installed as follows::
     $ git clone https://github.com/DigitalSlideArchive/HistomicsTK.git
     $ cd HistomicsTK
     # pip install git+https://github.com/cdeepakroy/ctk-cli
-    $ pip install --no-cache-dir -r requirements_c_conda.txt
-    $ pip install --no-cache-dir -r requirements.txt -r requirements_c.txt
+    $ pip install --no-cache-dir -r requirements.txt
```

Comment:
That is part of my plan! 

I tried adding it, but parse_requirements was throwing an error. So i am first making sure TravisCI succeeds without adding it to the requirements.txt file to isolate the issue. Then i will add it back and check.